### DISCUSSION TITLE
Advice on contributing PR to example apps

### DISCUSSION THREAD

**Discussion by erikdstock** [vercel/next.js#12319] [Help]
Hello folks - I've noticed there are few examples of next.js setups that include a jest configuration, so after doing a good amount of research I've made a PR with my own tooling setup. The PR is #12025, and I've made a few tweaks (removing references to husky + a conflicting eslint-prettier-config so it didn't need to be added to root) so far but am unsure how to move it forward as it still seems to be failing on a single azure pipeline and I have no idea how to debug it.

Apologies if this isn't the place to raise the question.

**Comment by timneutkens** [comment]
Unfortunately Azure pipelines is quite flaky and fails randomly sometimes. When changing examples it's fine to look at the GitHub actions results only given that we don't do any special checks on examples (only code style)

**Reply by erikdstock** [reply]
Thanks. Is there anything I can do to get this merged? I'd like to be able to use it myself on future projects.

**Reply by timneutkens** [reply]
We still need to review your PR, ever since 9.3 the amount of new PRs per day/week has gone up tremendously 🙏
